scripts/development/m3_stack/start_m3.sh

```diff
@@ -10,7 +10,21 @@ if [[ "$FORCE_BUILD" = true ]] ; then
 fi
 
 echo "Bringing up nodes in the background with docker compose, remember to run ./stop.sh when done"
+
+# need to start Jaeger before m3db or else m3db will not be able to talk to the Jaeger agent.
+if [[ "$START_JAEGER" = true ]] ; then
+    docker-compose -f docker-compose.yml up $DOCKER_ARGS jaeger
+    sleep 3
+    # rely on 204 status code until https://github.com/jaegertracing/jaeger/issues/1450 is resolved.
+    JAEGER_STATUS=$(curl -s -o /dev/null -w '%{http_code}' localhost:14269)
+    if [ $JAEGER_STATUS -ne 204 ]; then
+        echo "Jaeger could not start"
+        return 1
+    fi
+fi
+
 docker-compose -f docker-compose.yml up $DOCKER_ARGS m3coordinator01
+# docker-compose -f docker-compose.yml up --build -d --renew-anon-volumes m3db_seed
```

Comment:
does this need to be removed?

Reply:
Yes - sorry will remove before i land.

Reply:
cool, please put a `TODO(braskin)` or something else in case you're going to keep it around.